src/platform_impl/windows/mod.rs

```diff
@@ -248,3 +253,158 @@ impl Drop for PixelBuffer {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    #![cfg_attr(test, allow(dead_code))]
```

Comment:
Could you comment out the unused `GR_` constants rather than allowing dead code?

Reply:
Unused symbols have been commented out. I've also added a link to the winapi issue that tracks progress on the `GetGuiResources` API integration.